omtcmod.py

```diff
@@ -1,12 +1,15 @@
+import streamlit as st
 #welcome screen
 print('Hi, this program will convert your osu!mania scroll speed to etterna scroll speed')
 
 #asking for your data
-osu_mania_sp = int(input('\nPlease input your osu!mania Scroll Speed: '))
-etterna_recep = int(input('\nPlease input your Etterna Receptor Size: '))
+osu_mania_sp = st.number_input('\nPlease input your osu!mania Scroll Speed: ',
+                               min_value=1, max_value=40, step=1)
+etterna_recep = st.number_input('\nPlease input your Etterna Receptor Size: ',
+                               min_value=1, max_value=10000, step=10)

```

Comment:
I'm not sure what's the min max values for Receptor Size, will need your input for this